DOCS-530: Refactor Glossary (#996)
viamrobotics/docs: docs/appendix/glossary/frame.md

title: Frame
id: frame
full_link:
short_description: A frame represents a coordinate system that describes position and orientation of an object in relation to the Frame System.
aka:
---

> A frame represents a coordinate system that describes position and orientation of an object in relation to the {{< glossary_tooltip term_id="frame-system" text="Frame System" >}}.

Reviewer: ```suggestion
A frame represents a coordinate system that describes position and orientation of an object.
See also {{< glossary_tooltip term_id="frame-system" text="Frame System" >}}.
```
See comment above
